app/app.go

```diff
@@ -460,6 +477,11 @@ func NewLinkApp(
 		wasmOpts...,
 	)
 
+	fswapConfig := fswaptypes.DefaultConfig()
+	app.FswapKeeper = fswapkeeper.NewKeeper(appCodec, keys[fswaptypes.StoreKey], fswapConfig, app.BankKeeper)
+	const FinschiaV4BaseDenom = "cony"
+	app.FbridgeKeeper = fbridgekeeper.NewKeeper(appCodec, keys[fbridgetypes.StoreKey], memKeys[fbridgetypes.MemStoreKey], app.AccountKeeper, app.BankKeeper, FinschiaV4BaseDenom, fbridgetypes.DefaultAuthority().String())
```

Comment:
We should use another module address (i.e. foundation), because the version of the SDK does not support message based proposals.